### PR TITLE
Add devel_getBlockSyncPeers RPC API

### PR DESCRIPTION
**Pull request by majecty**

### CODE REVIEW

File: sync/src/block/mod.rs [hunk 1]
```diff
@@ -19,3 +19,9 @@ mod extension;
 mod message;
 
 pub use self::extension::Extension as BlockSyncExtension;
+
+use cnetwork::NodeId;
+
+pub trait BlockSyncForRPC: Send + Sync {
```
Comment (sgkim126) [inline sync/src/block/mod.rs]: How about renaming it to `BlockSyncInfo`

File: rpc/src/v1/impls/devel.rs [hunk 2]
```diff
@@ -82,4 +90,8 @@ where
         self.miner.stop_sealing();
         Ok(())
     }
+
+    fn get_block_sync_peers(&self) -> Result<Vec<SocketAddr>> {
+        Ok(self.block_sync.get_peers().iter().map(|node_id| node_id.clone().into_addr().into()).collect())
```
Comment (sgkim126) [inline rpc/src/v1/impls/devel.rs]:
```rust
Ok(self.block_sync.get_peers().into_iter().map(|node_id| node_id.into_addr().into()).collect())
```

File: sync/src/block/mod.rs (hunk 1 above)
Comment (majecty) [reply sync/src/block/mod.rs]: Good!

**Comment by majecty** [comment]
@jhs7jhs  I resolved the conflicts.
@sgkim126  I applied your comments.

File: rpc/src/v1/traits/devel.rs [hunk 3]
```diff
@@ -14,11 +14,12 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-use primitives::H256;
+use std::net::SocketAddr;
 
 use jsonrpc_core::Result;
 
 use super::super::types::Bytes;
+use primitives::H256;
```
Comment (sgkim126) [inline rpc/src/v1/traits/devel.rs]: Please fix the order

**Comment by majecty** [comment]
@sgkim126 I fixed it.

**Pull request merged**